calendar-bundle/src/Resources/contao/modules/ModuleCalendar.php

```diff
@@ -239,6 +239,9 @@ protected function compileWeeks()
 			$strClass = ($intCurrentDay < 2) ? ' weekend' : '';
 			$strClass .= ($i == 1 || $i == 8 || $i == 15 || $i == 22 || $i == 29 || $i == 36) ? ' col_first' : '';
 			$strClass .= ($i == 7 || $i == 14 || $i == 21 || $i == 28 || $i == 35 || $i == 42) ? ' col_last' : '';
+			
+			// Add timestamp to all cells
+			$arrDays[$strWeekClass][$i]['timestamp'] = strtotime(($intDay - 1) . ' day', $this->Date->monthBegin);
```

Comment:
Does this expression compile to e.g. `6 day`?

Reply:
Yes, or `-4 day` etc.

Reply:
I know `strtotime()` is very lax with its textual dates, but I didn't know that `6 day` is treated as `+6 days`. I assume you are sure about this? 😄

Reply:
Well I am already using it this way at least ;). With a custom module where I simply copy & pasted the compileWeeks() function and included the above change.